<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clicker</class>
 <widget class="QWidget" name="Clicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>56</width>
    <height>158</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <widget class="QCheckBox" name="cbDongle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>On</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbShift">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>45</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbCtrl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>124</y>
     <width>40</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ctrl</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbB1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>B1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbB2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>B2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbB3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>B3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbB4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>B4</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
